.eslintrc.js

```diff
@@ -18,6 +18,13 @@ module.exports = {
     sourceType: 'module',
   },
   plugins: ['react'],
+  settings: {
+    'import/resolver': {
+      node: {
+        extensions: ['.js', '.jsx', '.ts', '.tsx'],
+      },
+    },
+  },
```

Comment:
this was needed to get rid of errors for not including file extension in imports. with or without file extension, I was getting errors and this resolved it